etc/kayobe/environments/ci-multinode/inventory/group_vars/all/vxlan.yml

```diff
@@ -1,6 +1,7 @@
 ---
-vxlan_phys_dev: "eno1"
+vxlan_phys_dev: "{{ admin_oc_interface }}"
 vxlan_dstport: 4790
+vxlan_vni: 10
 vxlan_interfaces:
-  - device: vxlan10
+  - device: "vxlan{{ vxlan_vni }}"
     group: 224.0.0.10
```

Comment:
Looking at https://infohub.delltechnologies.com/l/planning-guide-vmware-cloud-foundation-3-x-on-vxrail/select-a-multicast-ip-address-range-for-vxlan-network It appears it would be to also change 224.0.0.0 to 239.0.0.0.

Could add something such as `group: {{ 239.0.0.0/8 | ansible.utils.next_nth_usable(vxlan_vni) }}`